logstash/pipeline/logstash.conf

```diff
@@ -1,7 +1,8 @@
 input {
-	tcp {
-		port => 5000
-	}
+    tcp {
+        port  => 5000
+        codec => json
```

Comment:
Not sure whether "json" is suitable for people who send raw lines, like in the `nc` example from the README.